packages/RichTextEditing-Core.package/RichTextStructureEditor.class/instance/buildStyleButtonPanelWith..st

```diff
@@ -0,0 +1,13 @@
+toolbuilder
+buildStyleButtonPanelWith: aBuilder
+
+	^ aBuilder pluggablePanelSpec new
+		model: self;
+		frame: (self styleButtonPanelFrame);
+		verticalResizing: #rigid;
+		children: {
+			self buildFontButtonWith: aBuilder.
+			self buildFontResetButtonWith: aBuilder.
+			self buildColorButtonWith: aBuilder.
+			self buildColorResetButtonWith: aBuilder};
```

Comment:
Incoherent use of closing bracket on newline

Reply:
I think this would be the correct way of formatting